src/Trace/Propagator/TraceContextPropagator.php

```diff
@@ -0,0 +1,70 @@
+<?php
+/**
+ * Copyright 2017 OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace OpenCensus\Trace\Propagator;
+
+use OpenCensus\Trace\SpanContext;
+
+/**
+ * This propagator uses HTTP headers to propagate SpanContext over HTTP.
+ * The default headers is `X-Cloud-Trace-Context`.
+ */
+class TraceContextPropagator implements PropagatorInterface
```

Comment:
What is this? Duplicate of HttpHeaderPropagator?

Reply:
removed this